packages/myst-frontmatter/src/licenses/licenses.yml

```diff
@@ -300,11 +347,21 @@ cases:
             my custom license but this is really long, like, more than one hundred characters long.
             clearly if it is this long it cannot be a name but must be a note.
     warnings: 1
-  - title: Other long string coerces to name (and warns)
+  - title: Other short string coerces to name (and warns)
     raw:
       license: my custom license
     normalized:
       license:
         content:
           name: my custom license
     warnings: 1
+  - title: Long string with "mit" buried in it is not coerced to MIT license
+    raw:
+      license: |
+        This is an open access article. Unrestricted non-commercial use is permitted provided the original work is properly cited
```

Comment:
The utility MyST uses to match license IDs with SPDX is _way_ too generous. It would pick up on the `mit` in the middle of "permitted" here, and coerce this to the MIT license. 😱